docs/blog/ibis-version-4.0.0-release.md

```diff
@@ -0,0 +1,41 @@
+# Ibis v4.0.0
+
+**by Patrick Clarke**
+
+09 January 2023
+
+## Introduction
+
+Ibis 4.0 has officially been released as the latest version of the package.
+This release includes several new backends, improved functionality, and some major internal refactors.
+A full list of the changes can be found in the project release notes [here](../release_notes.md).
+Let’s talk about some of the new changes 4.0 brings for Ibis users.
+
+## Backends
+
+Ibis 4.0 brings [Polars](https://ibis-project.org/docs/latest/backends/Polars/), [Snowflake](https://ibis-project.org/docs/dev/backends/Snowflake/), and [Trino](https://ibis-project.org/docs/dev/backends/Trino/) into an already-impressive stock of supported backends.
+The [Polars](https://www.pola.rs/) backend adds another way for users to work locally with DataFrames.
+The [Snowflake](https://www.snowflake.com/en/) and [Trino](https://trino.io/) backends add a free and familiar python API to popular data warehouses.
+
+Alongside these new backends, Google's BigQuery and mssql have been moved to the main repo and have been updated.
```

Comment:
What's the benefit of moving code to this repository? I understand the benefit it gives, but I don't know if the user who reads this text will also know it.